nuitka/utils/FileOperations.py

```diff
@@ -567,11 +567,15 @@ def copyTree(source_path, dest_path):
         This must be used over `shutil.copytree` which has troubles
         with existing directories.
     """
-
-    # False alarm on travis, pylint: disable=I0021,import-error,no-name-in-module
-    from distutils.dir_util import copy_tree
-
-    return copy_tree(source_path, dest_path)
+    try:
```

Comment:
Can you please make this a if `python_version >= 0x380` which will make it more explicit, what version it's about, catching exceptions is not very reliable and causes ugly and confusing tracebacks about causes in case the except handler fails.

Otherwise this seems perfectly fine solution.